user/src/com/google/gwt/user/server/rpc/impl/ServerSerializationStreamWriter.java

```diff
@@ -15,26 +15,29 @@
  */
 package com.google.gwt.user.server.rpc.impl;
 
-import com.google.gwt.user.client.rpc.CustomFieldSerializer;
-import com.google.gwt.user.client.rpc.SerializationException;
-import com.google.gwt.user.client.rpc.impl.AbstractSerializationStreamWriter;
-import com.google.gwt.user.server.Base64Utils;
-import com.google.gwt.user.server.rpc.SerializationPolicy;
-
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.ObjectOutputStream;
+import java.io.StringWriter;
+import java.io.Writer;
 import java.lang.reflect.Field;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
 import java.util.ArrayList;
 import java.util.IdentityHashMap;
+import java.util.LinkedList;
 import java.util.List;
-import java.util.ListIterator;
 import java.util.Map;
 import java.util.Set;
 
+import com.google.gwt.user.client.rpc.CustomFieldSerializer;
```

Comment:
i think the style check task will yell at you for moving imports like this

Reply:
9f934a903168a5509b5c2282882dd8c26986e9a9